proxy/nginx-common.conf

```diff
@@ -82,6 +82,6 @@ location = /500.html {
 }
 
 # prevent users from accessing: '/dataset/new' route, 'package_create' and 'resource_create' API routes
-location ~ ^/(dataset\/new|api\/action\/package_create|api\/action\/resource_create)\/? {
+location ~ ^/(dataset\/new|api\/action\/package_create|api\/action\/resource_create)/?$ {
```

Comment:
nice fix, fuhu!

Reply:
Did you really need to remove the `\`?  I like the end-of-line, easy fix 😉

Reply:
In nginx, escaping forward slash is opional. But since the leading `/` is not escaped, why escaping the trailing one.